dash_docs/chapters/dash_vtk/representations/index.py

```diff
@@ -0,0 +1,335 @@
+            - origin = (0,0,0)
+            - position = (0,0,0)
+            - scale = (1,1,1)
+            - visibility = 1
+            - pickable = 1
+            - dragable = 1
+            - orientation = (0,0,0)
+        - [__property__](https://kitware.github.io/vtk-js/api/Rendering_Core_ImageProperty.html):
+            - independentComponents = false
+            - interpolationType = [InterpolationType.LINEAR](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageProperty/Constants.js#L1-L4)
+            - colorWindow = 255
+            - colorLevel = 127.5
+            - ambient = 1.0
+            - diffuse = 0.0
+            - opacity = 1.0
+        - [__mapper__](https://kitware.github.io/vtk-js/api/Rendering_Core_ImageMapper.html):
+            - customDisplayExtent: [0, 0, 0, 0]
+            - useCustomExtents: false
+            - slice: 0
+            - slicingMode: [SlicingMode.NONE](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageMapper/Constants.js#L1-L9)
+            - closestIJKAxis: { ijkMode: [SlicingMode.NONE](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageMapper/Constants.js#L1-L9), flip: false }
+            - renderToRectangle: false
+            - sliceAtFocalPoint: false
+
+        ## PointCloudRepresentation
+
+        The __PointCloudRepresentation__ is just a helper using the following structure to streamline rendering a point cloud dataset. The code snippet below is not complete but it should provide you with some understanding of the kind of simplification that is happening under the hood.
+
+        ```python
+        def PointCloudRepresentation(**kwargs):
+        return dash_vtk.GeometryRepresentation(
+            id=kwargs.get('id'),
+            colorMapPreset=kwargs.get('colorMapPreset'),
+            colorDataRange=kwargs.get('colorDataRange'),
+            property=kwargs.get('property'),
+            children=[
+            dash_vtk.PolyData(
+                points=kwargs.get('xyz'),
+                connectivity='points',
+                children=[
+                dash_vtk.PointData([
+                    dash_vtk.DataArray(
+                    registration='setScalars',
+                    values={kwargs.get('scalars')}
+                    )
+                ])
+                ],
+            )
+            ],
+        )
+        ```
+
+        The set of convenient properties are as follows:
+        - __xyz__ = list of xyz of each point inside a flat array
+        - __colorMapPreset__ = color preset name to use
+        - __colorDataRange__ = rescale color map to provided that range
+        - __property__ = {} # Same as GeometryRepresentation/property
+        - __rgb__ / __rgba__ / __scalars__ = [...] let you define the field you want to color your point cloud with. The rgb(a) expects numbers up to 255 for each component: Red Green Blue (Alpha).
```

Comment:
```suggestion
        - __rgb__ / __rgba__ / __scalars__ = `[...]` let you define the field you want to color your point cloud with. The rgb(a) expects numbers up to 255 for each component: Red Green Blue (Alpha).
```
I don't think this was intended to be a link 😉